src/GenHDF5IO.chpl

```diff
@@ -0,0 +1,1630 @@
+module GenHDF5IO {
```

Comment:
It looks to me that you can compile Arkouda without adding a `use GenHDF5IO` in `arkouda_server.chpl`; is that correct? (assuming the function calls are removed from the `select` statement)

Reply:
See my other comment on `FileIO.chpl`.  The short answer is that `arkouda_server.chpl` still gets this dependency by way of `GenSymIO.chpl`.  It has the same issue that Parquet does.  It needs to be wired in somewhere and either mixing it into the command map or having a separate reader/write map/registry would be a way to unlink it (along with using a more general ArkoudaFileWriter/Reader class).